File: Client/src/app/entities/location.data.ts
import { V3 } from "./V3";
import * as THREE from 'three';
import { Group } from 'three/src/objects/Group.js';

export class LocationData {

  position!: V3;
  rotation!: V3;
  scale!: V3;

  mesh?: THREE.Group;
  markForDelete!: boolean;

}

File: Entities/LocationData.cs
namespace MG.Server.Entities
{
    public class LocationData: BaseData<LocationData>
    {
        public V3 Position { get; set; }
        public V3 Rotation { get; set; }
        public V3 Scale { get; set; }

        public LocationData()
        {
            Position = new V3();
            Rotation = new V3();
            Scale = new V3(1);
        }        

        

    }
}
